terminologies/technique/ultraHighFieldMagneticResonanceImaging.jsonld

```diff
@@ -0,0 +1,8 @@
+{
+ "@id":"https://openminds.ebrains.eu/controlledTerms/technique/ultraHighFieldMagneticResonanceImaging",

```

Comment:
I would not keep this one. We would keep "high field magnetic resonance imaging". Unless there is a clear definition between ultra-high-field and high-field.

@tgbugs: what do you think?